examples/basics/data_row_metadata.ipynb

```diff
@@ -3,7 +3,9 @@
   {
```

Comment:
<p>I think one of our initiatives from before was to remove empty spaces like this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/587/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>yeah ... I am okay with leaving it for now.</p><p> it will contain error messages if there is error occurs? </p><p> </p>